recipes/r-asics/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = 'ASICS' %}
+{% set version = '1.0.1' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
+  sha1: f0b6d0c44e26ee0350c9ec10e27d115c30307e63
+
+build:
+  number: 1
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, rmarkdown
+requirements:
+  build:
+    - r-base >3.6
```

Comment:
Anything that depends on `r-base` would fail with `Permission denied` error on Linux ARM64: https://github.com/bioconda/bioconda-recipes/pull/46177